lib/util/foretoken.ex

```diff
@@ -0,0 +1,22 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule Antikythera.Foretoken do
+  @moduledoc """
+  A thin wrapper around [`Foretoken`](https://github.com/skirino/foretoken) to avoid collisions of bucket name between executor pool.
```

Comment:
Let's briefly explain how we avoid collisions:

... avoid collisions between bucket names by prefixing executor pool IDs.

Reply:
Fixed in 1019fb721e18ae6f481e67446d31292a70595834